terminus-ui/select/src/select.component.ts

```diff
@@ -73,25 +72,19 @@ import {
   takeUntil,
 } from 'rxjs/operators';
 
-
-import {
-  TsAutocompletePanelComponent,
-  TsAutocompletePanelSelectedEvent,
-} from './autocomplete/autocomplete-panel.component';
-import { TsAutocompleteTriggerDirective } from './autocomplete/autocomplete-trigger.directive';
-import { TsSelectOptgroupComponent } from './optgroup/optgroup.component';
+import { TsOptgroupComponent } from '@terminus/ui/option';
 import {
   allOptionsAreSelected,
   countGroupLabelsBeforeOption,
   getOptionScrollPosition,
   someOptionsAreSelected,
   toggleAllOptions,
-} from './option/option-utilities';
+} from '@terminus/ui/option';
```

Comment:
I actually thought about it, but wasn't totally sure it's internal only module. Is there any chance that this might be exposed to consumers? If not, we probably don't want to export the module then.

Reply:
I’m pretty sure ngPackagr requires us to export everything. 

I can’t think of a reason someone would need an option by itself. But if they did, they could import it from the top level.